app/Http/Controllers/SampleController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use Illuminate\Support\Facades\Mail;
+use App\Mail\SampleNotification;
+
+class SampleController extends Controller
```

Comment:
Please avoid using non informative class name such us SampleController, TestController and etc.